screwdriver/screwdriver.go

```diff
@@ -270,6 +271,10 @@ func (a api) put(url *url.URL, bodyType string, payload io.Reader) ([]byte, erro
 	return a.write(url, "PUT", bodyType, payload)
 }
 
+func (a api) GetAPIURL() string {
+	return a.baseURL
```

Comment:
Will anyone who deploys their own Screwdriver cluster also have the v4 at the end of their API URL?